wlauto/workloads/adobereader/uiauto/app/src/main/java/com/arm/wlauto/uiauto/adobereader/UiAutomation.java

```diff
@@ -102,6 +102,10 @@ private void dismissWelcomeView() throws Exception {
         if (!onboarding_finish_button.exists()) {
             welcomeView.swipeLeft(10);
         }
+        
+        if (!onboarding_finish_button.exists()) {
+        return;
+        }
```

Comment:
Makes sense. Will change my implementation, drop this commit and have another to you shortly.